### PR TITLE
17769 acct settings loa1 style guide changes

### PR DESCRIPTION
**Pull request by tlei123**
## Testing done
17769 - Change Account Settings LOA1 content per Style Guide updates.  See [changes doc](/department-of-veterans-affairs/vets.gov-team/files/3040584/Acct.Settings-LOA1_Content.updates_Feb.2019.docx) for details.

## Acceptance criteria
- [ ] All content-changes made per [changes doc.](/department-of-veterans-affairs/vets.gov-team/files/3040584/Acct.Settings-LOA1_Content.updates_Feb.2019.docx)

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/account/components/AccountVerification.jsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export default function AccountVerification({ loa }) {
               status or send secure messages to your health care team?
             </h4>
             <p>
-              We need to make sure you’re you-and not someone pretending to be
-              you-before we can give you access to your personal and
+              We need to make sure you’re you—and not someone pretending to be
```
Comment (erikphansen) [inline src/applications/personalization/account/components/AccountVerification.jsx]: These don't look like em dashes to me :) `—`

Comment (tlei123) [reply src/applications/personalization/account/components/AccountVerification.jsx]: It's a dumb text-editor display thing (incl. browser dev-tools too); browser webpage displays em-dashes properly:
17769-em-dash-display

**Comment by tlei123** [comment]
Just merged latest master; now watching those tests.

**Comment by tlei123** [comment]
@erikphansen , see my Comment above re. the em-dash display-quirks.  Pls. Resolve the Conversation once you're satisfied, and Approve PR.  Thx!

All checks have now passed.

**Review by erikphansen — APPROVED** [review]
Sorry about doubting your ability to insert the right glyph :-p

**Pull request merged**